### DISCUSSION TITLE
Desactivate Copilot by default

### DISCUSSION THREAD

**Discussion by glcraft** [community/community#7191] [Copilot Conversations] [answered]
Actually, we can manually deactivate copilot by clicking on the button in the status bar of vscode.
Because of possible sensitive work, it would be nice to manually enable copilot instead of toggling it based on the language.
I was thinking of an option in the settings. That way, we could enable copilot by default then disable it on certain workspace (or vice versa)

**Comment by Krzysztof-Cieslak** [comment, marked as answer]
There exists a setting for that. 
You can put `"github.copilot.enable": { "*": false },` in your `settings.json` file (either in workspace settings, or global settings) 

I do recommend checking our configuration documentation here - https://github.com/github/copilot-docs/blob/main/docs/visualstudiocode/configuring.md

**Comment by giovannibenussi** [comment]
If you want to disable Copilot on vim, you can add the following to your `.vimrc` file:

```
" Disable Copilot by default
let g:copilot_enabled = v:false
```
